.gitignore

```diff
@@ -1,9 +1,6 @@
-modules/utils/src/version_impl.h
```

Comment:
I don't think these changes here are intentional or there is a good reason to have them on our side